example/webhooks/v1/request.proto

```diff
@@ -0,0 +1,91 @@
+syntax = "proto3";
+
+package webhooks.v1;
+
+import "apigw/v1/apigw.proto";
+import "google/protobuf/any.proto";
+
+option go_package = "gitlab.com/ductone/c1/pkg/pb/c1/webhooks/v1beta";
+
+message Body {
+  option (apigw.v1.message).message_options = {force_expose: true};
```

Comment:
shouldn't this be a Webhook annotation?  because in the OAI spec, there are webhook specific sections right?

Reply:
like might expect something more like this:
```
option (apigw.v1.message).message_options = {webhook_request: "NameOfWebhook"};
```

```
option (apigw.v1.message).message_options = {webhook_response: "NameOfWebhook"};
```

And this generates a named webhook block in the OpenAPI Spec?

Reply:
Neat, let me look into how this would change to do that.

Reply:
Is this along the lines of what you had in mind? We sorta have this unified webhook experience that I don't think fit well with an openapi spec. Like we can set the name but is that name valuable? https://github.com/ductone/protoc-gen-apigw/pull/50/commits/10f64b671fe80a963937d6aeb1a05ce3fadebd5e